TeamCode/src/main/java/org/firstinspires/ftc/teamcode/components/TeleOpMovement.kt

```diff
@@ -39,9 +40,15 @@ object TeleOpMovement : Component {
         // angle and strength
         // PI / 3 because 0 radians is right, not forward
         val joyRadians = atan2(joyY, joyX) - (PI / 3.0)
-        val joyMagnitude = sqrt(joyY * joyY + joyX * joyX)
+        var joyMagnitude = sqrt(joyY * joyY + joyX * joyX)
 
-        // all movement of the wheels
+        // if the right bumper is pressed move slower
+        if (gamepad.right_bumper) {
+            joyMagnitude *= SLOW_MULTIPLIER
+        }
+
+        // movement of all wheels
         TriWheels.driveWithRotation(joyRadians, joyMagnitude, rotationPower)
+        }
```

Comment:
This fails to compile, but I'm committing a fix to remove this extra parentheses.